.github/workflows/update_dependabot_prs.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Customize PR body 1
```

Comment:
Can we make this something more descriptive?

Reply:
yes